src/beamlime/applications/handlers.py

```diff
@@ -530,7 +531,7 @@ def _plot_2d(
         aspect = 'equal'
 
     values = da.values
-    ax.imshow(values, **kwargs, aspect=aspect)
+    image = ax.imshow(values, **kwargs, aspect=aspect)
```

Comment:
Are you missing a `origin='lower'` to have the intuitive image layout (instead of the web layout)?

Reply:
Ah yes! I missed that :-)